scripts/nolints.json

```diff
@@ -1,4 +1,7 @@
-[["docBlame", "AddGroupWithOne"],
+[["defLemma", "List.Lex.isOrderConnected.aux"],
+ ["defLemma", "List.Lex.isTrichotomous.aux"],
+ ["defLemma", "List.Lex.isAsymm.aux"],
+ ["docBlame", "AddGroupWithOne"],
```

Comment:
It would be better if these were explicit `@[nolint]` attributes (with a link to the lean4 PR that will make them obsolete.

@[nolint] is for cases where there is a reason to ignore the linter. `scripts/nolints.json` is for embarrassing historical accidents.